tfutils/base.py

```diff
@@ -274,76 +318,47 @@ def load_rec(self):
                                      collfs=self.load_collfs,
                                      collfs_recent=self.load_collfs_recent)
             if load is None:
-                raise Exception('You specified load parameters but no record was found with the given spec.')
+                raise Exception(
+                    'You specified load parameters but no record was found with the given spec.')
         self.load_data = load
 
-    def initialize(self):
-        """
-        Fetches record then uses tf's saver.restore
-        """
+    def initialize(self, no_scratch=False):
```

Comment:
@chengxuz can you and alex chat about *why* this is being done and make sure it's a clean and simple use case?  and not covered / interacting with other things?